examples/redis-scaler/Readme.md

```diff
@@ -0,0 +1,102 @@
+## Scenario
+
+This example realises scaling of jobs based on the size of [list](https://redis.io/docs/data-types/lists/) in redis.
+Each job, when scheduled, is poping a message from the list and stops.
+Additionally there is a function that awaits  HTTP trafic and populates messages to redis list.
+
+![diagram](assets/scenario.drawio.svg)
+
+## Prerequisistes
+
+ - `KUBECONFIG` env variable pointing to a kubernetes instance
+ - helm client installed locally in your machine
+ - keda and serverless modules enabled in th kyma cluster (`make ensure_kyma_modules`)
+
+
+## Deploy 
+
+### Deploy redis
+
+Install redis into `redis` namespace.
+
+```sh
+kubectl create ns redis
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo update
+helm install my-release bitnami/redis --namespace redis
+```
+
+### Deploy example
+
+```sh
+make deploy_app
+```
```

Comment:
Please add three spaces in line 33. When I try to add it as a suggestion, they disappear.